src/ophyd_async/panda/panda.py

```diff
@@ -86,6 +87,27 @@ class SeqBlock(Device):
     table: SignalRW[SeqTable]
 
 
+@dataclass
+class SeqTableRow:
```

Comment:
This is inspired by @tpoliaw's tables.py, but I chose dataclasses instead of namedtuples for printability and typing

Reply:
Should we use TypedDict for consistency with SeqTable?

Reply:
Should the table functions/types be moved to panda/tables.py, to keep it a little easier to read?

Reply:
> Should we use TypedDict for consistency with SeqTable?

The only issue with this is the lack of defaults. `SeqTable` is a fully populated output to go to the PandA. It can be constructed with `seq_table_from_arrays` to handle consistent lengths. If we made the `SeqTableRow` a `TypedDict` then we'd have to add the defaults in `seq_table_from_rows`. Not sure which I prefer. Thoughts?

Reply:
Have moved and exported from package but left as is in the interest of time.